docker/templates/new-django-app/{{cookiecutter.app_name}}/webpack.config.js

```diff
@@ -0,0 +1,83 @@
+const path = require("path")
+const webpack = require("webpack") // eslint-disable-line no-unused-vars
+const BundleTracker = require("webpack-bundle-tracker")
+
+const config = {
+  context: __dirname,
+  entry: {
+    base: "./{{cookiecutter.app_name}}/static/js/base.js",
```

Comment:
I think we should be subbing in the module name rather than the app name:

```suggestion
    base: "./{{cookiecutter.module_name}}/static/js/base.js",
```

Reply:
good catch, ty